asyncwrapper/src/main/kotlin/org/n/riesgos/asyncwrapper/config/RetryConfiguration.kt

```diff
@@ -0,0 +1,7 @@
+package org.n.riesgos.asyncwrapper.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+
+data class RetryConfiguration(var attempts: Int = 3, var backoff_millis: Long = 3000)
```

Comment:
Also the question: Do we need to have those fields modifiable (`var`)? I would favor immutability (`val`) whereever this is possible.